llvm/lib/Target/RISCV/RISCVSchedSiFive7.td

```diff
@@ -937,8 +937,9 @@ foreach mx = SchedMxList in {
   defvar Cycles = SiFive7GetCyclesDefault<mx>.c;
   defvar IsWorstCase = SiFive7IsWorstCaseMX<mx, SchedMxList>.c;
   let Latency = 4, AcquireAtCycles = [0, 1], ReleaseAtCycles = [1, !add(1, Cycles)] in {
-    defm "" : LMULWriteResMX<"WriteVISlideX",   [SiFive7VCQ, SiFive7VA], mx, IsWorstCase>;
```

Comment:
The spaces between `,` and `[` mismatched, should we make them consistent?

Reply:
It's fixed now.